students/style.css

```diff
@@ -0,0 +1,52 @@
+* {
```

Comment:
💭 Love this file. I'd recommend making sure that the spacing between all of the CSS looks consistent though:

```
 .css {
       property: value;
       property: value;
   }
```